content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
+
+The goal of cycle counts is to count more often at key locations to keep your critical stock levels
+accurate.
+
+Configuration
+=============
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
+the storage location. That also means that to use cycle counts you need to activate the setting
```

Comment:
- Replace the sentence on Lines 16-17 `That also means...**Storage Locations**.` with `To activate storage locations, go to :menuselection:'Inventory --> Configuration --> Settings --> Warehouse' and activate the :guilabel:'Storage Locations' setting.`